aws/data_source_aws_cur_report_definition_test.go

```diff
@@ -11,12 +11,10 @@ import (
 )
 
 func TestAccDataSourceAwsCurReportDefinition_basic(t *testing.T) {
-	if testAccGetPartition() != "aws" {
-		t.Skipf("skipping test; hardcoded us-east-1 test, not supported in partition %s", testAccGetPartition()) //lintignore:AWSAT003
-	}
+	testAccPartitionHasServicePreCheck("cur", t) // checked before env vars cause creds/prechecks to fail
```

Comment:
qq re: the comment here (and other places) -- suggests the tests should fail since this line occurs before the env vars/test precheck below but test output looks good , am I reading this right 😅 ?

Reply:
Yeah, I was not very clear with that. I changed it - I think it makes more sense now.

Reply:
gotcha! yep looks goood 👍